### PR TITLE
chore(designer): Clean Up Some Custom Editor Code

### PR DESCRIPTION
**Pull request by Eric-B-Wu**
Not used by Logic Apps, but there were some warnings in the files that custom editors were using, so I did a mini cleanup. There will be more cleanup we'll need to do in the future, but this also gives a starting point for potential custom editors we have in the future.

Also mini change to adding action "HTTP + Swagger" will now be named that instead of "httpwithswagger"

### CODE REVIEW

File: pnpm-lock.yaml
Comment (ccastrotrejo) [inline pnpm-lock.yaml:1]: NIT: is this change needed?

**Review by ccastrotrejo — APPROVED** [review]

File: pnpm-lock.yaml
Comment (Eric-B-Wu) [reply pnpm-lock.yaml:1]: we were having some errors being thrown due to type issues in vitest so this should fix those errors
